### ISSUE TITLE
Track-list play control doesn't resume paused play, it restarts track

### ISSUE THREAD

**Issue by prestonaustin** [therabble/ccr-app#56]
It should resume play as the mini-player control does.
(Preston's Firefox on Mac OS X, http://wax.capitalcityrecords.ca/albums/the-duettes-your-little-voice, 10 Jul, 9:05c)

**Comment by prestonaustin** [comment]
@acecarruthers I'm going to tag "would be better if this didn't happen this way" bugs like this non-blocking, and "I can't do what I'm here for" bugs blocking - just FYI.

**Comment by prestonaustin** [comment]
@mbrakken Do you think this is linked to other issues with player state updates? An opinion on where to start here would be good.

**Comment by mbrakken** [comment]
No. Two completely different actions/methods are called, since clicking a
track passes an album & track to the player while clicking the player
buttons just interact with the player.

**Comment by mbrakken** [comment]
The [track] (https://github.com/therabble/ccr-app/blob/master/app/components/y-track.js#L14) handles the click on the track and sends the "play" action. Based on the [template] (https://github.com/therabble/ccr-app/blob/master/app/templates/albums/album.hbs#L48), this calls [`playPause()`] (https://github.com/therabble/ccr-app/blob/master/app/templates/albums/album.hbs#L48) in the application route.
```javascript
playPause: function (track, evt){
    // ga('send','event','Streaming','play',track,1);
    var player = this.get('player');
    if (track.get('playing')) {
        track.pause();
    } else {
        player.play(track, evt);
    }
}
```
This in turn calls [`play()`] (https://github.com/therabble/ccr-app/blob/master/app/services/player.js#L34) in the player
```javascript
play: function(track, evt){
    var content = this.get('content');
    if (typeof track !== 'undefined') {
        if (content !== null && content !== undefined) {
            content.stop();
        }
        this.set('content', track);
        this.get('content').play(evt);
    }
}
```
Comparing this with the `toggle()` method in the player:
```javascript
toggle: function(){
    var content = this.get('content');
    if (content.get('playing')) {
        content.pause();
    }
    else {
        content.resume();
    }
}
```
I think we could compare the track set on the `content` attribute for the player, and if the track === the content, we could call `track.resume()` instead of `this.set('content', track); this.get('content').play(evt);`.

It'd look something like this (untested):
```javascript
// play() in player.js
play: function(track, evt){
    var content = this.get('content');
    if (typeof track !== 'undefined') {
        if (track === content) {
            track.resume();
        } else {
            if (content !== null && content !== undefined) {
                 content.stop();
            }
            this.set('content', track);
            this.get('content').play(evt);
        }
    }
}
```

**Comment by mbrakken** [comment]
will be closed by #77 (implementation of the above code block)

**Comment by mbrakken** [comment]
reassigned for test/merge/deploy.
